### DISCUSSION TITLE
SonOff consumption data

### DISCUSSION THREAD

**Discussion by CoyoteProd** [dawidchyrzynski/arduino-home-assistant#96] [Q&A]
More a question than an issue :

It is possible to get SonOff consumption data and push it to MQTT Broker for to be used in Home Assistant ?

Thanks a lot
Vil

**Comment by dawidchyrzynski** [comment]
Hi @CoyoteProd,

As long as you can code business logic (measuring power consumption) for Sonoff then it's possible. This library only provides API for communication with the HA. Everything else is out of its scope.
